dom0/sd-workstation.top

```diff
@@ -38,15 +40,12 @@ base:
     - sd-sys-firewall-files
```

Comment:
Good idea. I've added a config test for this case. Since a few of the VMs customize the rsyslog config on boot, via rc.local, there may be a race where the log messages you identify can still slip in. Currently, the config test expects zero occurrences. If it turns out one or two slip in, we can update the logic to be <=2 or so.